assets/app/actions/siteActions.js

```diff
@@ -55,6 +55,23 @@ export default {
     }).catch(error => alertActions.httpError(error.message));
   },
 
+  uploadFile(site, path, fileData, message) {
```

Comment:
Also, I don't think the `createCommit` action should redirect. I think it should return a promise and then the consumer of `createCommit` can decide if it should redirect or not. I think we just did this in our pairing session...